app/views/support/schools/opt_outs/edit.html.erb

```diff
@@ -0,0 +1,26 @@
+<%- title = t('page_titles.support.schools.opt_outs.edit') %>
+<%- content_for :title, title %>
+<%- content_for :before_content do %>
+  <%= govuk_link_to('Back', support_school_path(@school), class: 'govuk-back-link') %>
+<%- end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <span class="govuk-caption-xl"><%= @school.name %></span>
+      <%= title %>
+    </h1>
+
+    <%= form_with model: Support::OptOutForm.new(school: @school), url: support_school_opt_out_path, method: :put, scope: :school do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <% opt_out_options = [OpenStruct.new(id: 1, name: 'Yes, opt out and no longer receive communications'), OpenStruct.new(id: 0, name: 'No, opt in and receive communcations')] %>
```

Comment:
```suggestion
      <% opt_out_options = [OpenStruct.new(id: 1, name: 'Opt out and no longer receive communications'), OpenStruct.new(id: 0, name: 'Opt in and receive communications')] %>
```

(Note there's a type on the no option)

Reply:
The "yes, opt out" is a little odd

Reply:
The yes/no bit has gone and screenshot has been updated